integration_tests/snapshots/logs/http-requests_node12.log

```diff
@@ -14,8 +14,8 @@ START
   ],
   "v": 1
 }
-HTTP GET https://ip-ranges.datadoghq.com/ Headers: ["Accept:application/json, text/plain, */*","User-Agent:axios/X.X.X","x-datadog-parent-id:XXXX","x-datadog-sampling-priority:-1","x-datadog-trace-id:XXXX"]
-HTTP GET https://ip-ranges.datadoghq.eu/ Headers: ["Accept:application/json, text/plain, */*","User-Agent:axios/X.X.X","x-datadog-parent-id:XXXX","x-datadog-sampling-priority:-1","x-datadog-trace-id:XXXX"]
+HTTP GET https://ip-ranges.datadoghq.com/ Headers: ["Accept-Encoding:gzip, deflate, br","Accept:application/json, text/plain, */*","User-Agent:axios/X.X.X","x-datadog-parent-id:XXXX","x-datadog-sampling-priority:-1","x-datadog-trace-id:XXXX"]
```

Comment:
The biggest problem with this PR is that I'm not very sure how this got updated.